_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,407 @@
+---
+id: 9bd38c
+name: Non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when there is a visual reference of content, there are also non-visual indicators of the location.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  -  DOM Tree
+  -  Language
+acknowledgements:
+  authors:
+    -  Brian Bors
+    -  Daniël Strik
+    -  Wilco Fiers
+---
+
+## Applicability
+
+Any text node that includes one of the [visual reference words](#visual-reference-words), that is [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree)
+
+## Expectation
+
+Each target that describes any [web content](https://www.w3.org/TR/WCAG21/#dfn-content) through the use of the [visual reference words](#visual-reference-words), is included in a [section of the content](#section-of-content) that also describes that [web content](https://www.w3.org/TR/WCAG21/#dfn-content) by a non-visual property, except if the target is not part of an instruction about [web content](https://www.w3.org/TR/WCAG21/#dfn-content) or the visual reference word is included in the described content.
```

Comment:
Thank you for the questions:

- Describes is indeed subjective, but not ambiguous. A text alternative describes an image for example. There is a duplication in the expectation that we shall fix (see reply above).
- See the note below. We will add website.
- This is the duplication that we shall remove.
- We will rename this to a "text instruction" (because we are only talking about text nodes)
- All of the above. This is mentioned in the definition of web content.